Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8
+
+COPY . /app
+WORKDIR /app
+RUN apt update && apt install -yqq g++ gcc libc6-dev make pkg-config libffi-dev python3-dev git
+RUN pip3 install pipenv
+RUN pipenv install --system --deploy --ignore-pipfile
+RUN pip3 install -r requirements.txt
+RUN chmod +x /app/run_linux.sh
+RUN rm /app/.env ## we don't need this
```

Comment:
How are you passing the credentials into the container for the bot to run? I don't see envars being used in the docs file that was created.

Reply:
Oops, i fixed this in my latest commit.  The user just needs to update the .env.example and rename to .env and then build the image.